ai/STA/Tactic/position_for_pass.py

```diff
@@ -95,7 +96,7 @@ def _find_best_player_position(self):
             d = enemy.position - center
             # Clamp distance norm
             d = MIN_DIST_FROM_CENTER * normalize(d) if d.norm < MIN_DIST_FROM_CENTER else d
-            # Square of the inverse of the distance, a bit like Newton's law of universal gravitation
+            # Cube of the inverse of the distance as magnets repulsing each other.
```

Comment:
It's the square, not the cube. Since the d is both normalize and divided by the inverse square of the distance